force_gromacs/data_sources/fragment/fragment_data_source.py

```diff
@@ -0,0 +1,51 @@
+from force_bdss.api import BaseDataSource, DataValue, Slot
+
+from force_gromacs.io.gromacs_topology_reader import (
+    GromacsTopologyReader
+)
+
+from .fragment import Fragment
+
+
+class FragmentDataSource(BaseDataSource):
+    """Class takes in all data required to define each
+    separate molecular fragment in a Gromacs experiment. Gromacs topology
+    files must be included for all species, however coordinate files
+    are not necessary for atoms or molecules represented by single beads.
+    """
+
+    # --------------------
+    #  Regular Attributes
+    # --------------------
+
+    #: Topology reader that can be used to parse '.itp' files. Currently
+    #: unused, but can be referred to by any subclass that would like to
+    #: provide additional pre-processing functionality.
+    reader = GromacsTopologyReader()
```

Comment:
Unused `reader` attribute, could make private or remove as file parsing now occurs inside the `Fragment` class